public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "Configurer le Duché souterrain",
+      "setup": "<i>1.</i> Choisissez une clairière de patrie qui n'est pas adjacente aux patries ennemies.<br/><i>2.</i> Mettez 2 <strong>guerriers</strong> et 1 <strong>tunnel</strong> dans votre pays d'origine. Placez 5 <strong>guerriers</strong> parmi les clairières adjacentes aussi uniformément que possible.<br/><i>3.</i> Placez votre <strong>Panneau de Terrier< /strong> près de la carte. Remplissez vos pistes Bâtiments avec vos <strong>citadelles</strong> et <strong>marchés</strong>. Mettez vos 9 <strong>cartes de ministre</strong> sur votre pile de Ministres Insoumis. Mettez vos 9 <strong>couronnes</strong> sur les cases carrées indiquant les points de victoire."
+    },
+    "eyrie": {
+      "name": "Dynasties Eyrie",
+      "summaryTitle": "Jouer les Dynasties Eyrie",
+      "summary": "<p>En tant que dynasties Eyrie, vous souhaitez redonner à votre espèce autrefois digne sa gloire d'antan en reprenant le contrôle de la forêt. À chaque tour, vous marquez des points pour vos <strong>perchoirs</strong> sur la carte. plus de perchoirs, plus vous marquez de points.</p><p>Cependant, vous êtes lié par le <strong>décret</strong>, un mandat du <strong>chef</strong> Eyrie. À chaque tour, vous doit ajouter des cartes au décret, puis effectuer une action pour chaque carte qu'il contient. Chaque action doit se produire dans la clairière correspondant à sa carte, alors planifiez judicieusement. Au début, c'est simple, mais au fur et à mesure que le décret passe à 10 ou 12 cartes, vous vous retrouverez à vous démener pour terminer chaque action. Si vous ne pouvez pas en terminer une, vous tomberez dans <strong>la tourmente</strong>, vous perdrez des points, remplacerez votre chef et annulerez le décret.</p> <p>Avant tout, rappelez-vous : les créatures du sol tremblent et se dispersent à votre arrivée. Vous êtes les <strong>Seigneurs de la Forêt</strong> - vous dirigez une clairière même lorsque vous êtes lié à la présence. Cependant, votre peuple a le <strong>dédain pour le commerce</strong>, de sorte que vous marquerez souvent moins de points pour la fabrication d'objets.</p>",
+      "setupTitle": "Configurer les Dynasties Eyrie",
+      "setup": "<i>1.</i> Choisissez une clairière de patrie sur la clairière du bord de la carte qui a 2+ clairières entre elle et les patries ennemies.<br/><i>2.</i> Mettez 6 <strong>guerriers< /strong> et 1 <strong>perchoir</strong> dans votre patrie.<br/><i>3.</i> Placez n'importe quelle <strong>carte de chef</strong> sur votre emplacement de carte de chef. Gardez l'autre 3 chefs face visible à proximité.<br/><i>4.</i> Placez vos 2 <strong>Vizirs loyaux</strong> sous les emplacements de la colonne Décret comme indiqué sur votre chef actuel.<br/><i> 5.</i> Remplissez vos espaces Perchoirs avec des <strong>perchoirs</strong>, sauf le plus à gauche."
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.